burger/src/components/Faq/Faq.js

```diff
@@ -0,0 +1,18 @@
+export default function Faq() {
+  return (
```

Comment:
There was a proposal in the task on how to style the content of the page, furthermore there are a lot of ready-to-use options in the UI library, and simple HTML elements look as too easy solution